AFQ/recognition/criteria.py

```diff
@@ -344,6 +330,12 @@ def run_bundle_rec_plan(
         mapping,
         img.affine,
         apply_to_recobundles=True))
+    apply_to_roi_dict(
```

Comment:
If you reverse what is and isn't a method above, I think the call here would become 

```
bundle_dict.apply_to_roi(bundle_def, _roi_distance_helper, ...)
```